### PR TITLE
PLT-3191 E20: Updates to Reload Configuration from Disk and Recycle Database Connections 

### PR DESCRIPTION
**Pull request by DavidLu1997**
Added help text to `Configuration > Reload Configuration from Disk` and `Database > Recycle Database Connections` per PM spec. Moved `Configuration > Reload Configuration from Disk` to the bottom of its page (as it is rarely used).

image

image

### CODE REVIEW

**Comment by coreyhulen** [comment]
+1

**Comment by enahum** [comment]
+1

File: webapp/components/admin_console/reload_config.jsx [hunk 1]
```diff
@@ -64,6 +64,13 @@ export default class ReloadConfigButton extends React.Component {
             );
         }
 
+        let helpText = (
+            <FormattedHTMLMessage
+                id='admin.reload.reloadDescription'
+                defaultMessage='Deployments using multiple database can switch from one master database to another without restarting the Mattermost server by updating "config.json" to the new desired configuration and using the <b>Reload Configuration from Disk</b> feature to load the new settings while the server is running. The administrator should then use the <a href="../advanced/database"><b>Database > Recycle Database Connections</b></a> feature to recycle the database connections based on the new settings.'
```
Comment (it33) [inline webapp/components/admin_console/reload_config.jsx]: should be "multiple database**s**"?

File: webapp/i18n/en.json [hunk 2]
```diff
@@ -387,10 +387,12 @@
...
   "admin.regenerate": "Regenerate",
   "admin.reload.button": "Reload Configuration From Disk",
   "admin.reload.loading": " Loading...",
   "admin.reload.reloadFail": "Reloading unsuccessful: {error}",
+  "admin.reload.reloadDescription": "Deployments using multiple database can switch from one master database to another without restarting the Mattermost server by updating \"config.json\" to the new desired configuration and using the <b>Reload Configuration from Disk</b> feature to load the new settings while the server is running. The administrator should then use the <a href=\"../advanced/database\"><b>Database > Recycle Database Connections</b></a> feature to recycle the database connections based on the new settings.",
```
Comment (it33) [inline webapp/i18n/en.json]: should be "multiple database**s**" (plural)?

**Comment by DavidLu1997** [comment]
Typos fixed

**Pull request merged**